doc/developer/platform/ux.md

```diff
@@ -772,6 +772,15 @@ running will be hidden from users. Any log messages in `materialized` today that
 are meant for end-users will need to instead make their messages available via
 tables or sources in the system catalog.
 
+### Correctness
+
+Materialize aims to maintain three correctness guarantees.
```

Comment:
I am not 100% sure the tense we want in this doc, but "Materialize maintains three correctness guarantees:" might be better?

Reply:
Yes!